recipes/zoe/all/conandata.yml

```diff
@@ -0,0 +1,17 @@
+sources:
+  "3.0":
+    url: "https://github.com/winsoft666/zoe/archive/refs/tags/v3.0.tar.gz"
+    sha256: "718c6de993bea2a4b1866070a8685e77cfa7929bbd9e22abb29e54c67a687a5e"
+
+patches:
+  "3.0":
```

Comment:
Do you plan to submit the other two patches upstream?

Reply:
Thanks!
I am going to create PR for two.